src/main/kotlin/no/nav/familie/ef/mottak/service/FeltformatererPdfKvittering.kt

```diff
@@ -19,7 +19,8 @@ object FeltformatererPdfKvittering {
      */
     fun genereltFormatMapperMapEndenode(entitet: Søknadsfelt<*>): VerdilisteElement? {
         // skal ekskluderes
-        val skalEkskluderes = ((entitet.label == "Jeg har sendt inn denne dokumentasjonen til Nav tidligere" || entitet.label == "I have already submitted this documentation to Nav in the past") && entitet.verdi.toString() == "false") ||
+        val skalEkskluderes =
+            ((entitet.label == "Jeg har sendt inn denne dokumentasjonen til Nav tidligere" || entitet.label == "I have already submitted this documentation to Nav in the past") && entitet.verdi.toString() == "false") ||
                 (entitet.label == "Født" && entitet.verdi.toString() == "true")
```

Comment:
Hva om vi bryter det litt mer ned?
```suggestion
        val erTidligereInnsendt = (entitet.label == "Jeg har sendt inn denne dokumentasjonen til Nav tidligere" || entitet.label == "I have already submitted this documentation to Nav in the past") && entitet.verdi.toString() == "false"
val erFødt = entitet.label == "Født" && entitet.verdi.toString() == "true"

val skalEkskluderes = erTidligereInnsendt || erFødt
```

Reply:
Ta gjerne med den engelsker versjonen av: "entitet.label == "Født"

Reply:
eller finnes det en engelsk versjon av den søknaden?

Reply:
dette er formattering. så refaktorering av denne kan vell gjøre si en annen PR